language/en/common.php

```diff
@@ -113,6 +113,9 @@
 	'SKELETON_QUESTION_COMPONENT_NOTIFICATION'		=> 'Create a sample notification?',
 	'SKELETON_QUESTION_COMPONENT_NOTIFICATION_UI'	=> 'Notifications',
 	'SKELETON_QUESTION_COMPONENT_NOTIFICATION_EXPLAIN'	=> 'Notifications allow an extension to notify users of specific activities.',
+	'SKELETON_QUESTION_COMPONENT_PERMISSIONS'		=> 'Create sample permissions?',
+	'SKELETON_QUESTION_COMPONENT_PERMISSIONS_UI'	=> 'Permissions',
+	'SKELETON_QUESTION_COMPONENT_PERMISSIONS_EXPLAIN'	=> 'Permissions are used to grant users, groups and roles access to extension features.',
```

Comment:
I feel like this sounds to 'implicit'. While other lines say "Can be used" and "Allow you to" structures, this one implies that extension features are otherwise not available.
Perhaps something more along the lines of `Permissions can be used to grant/deny users, ...`